### PR TITLE
chore: update deps (including security updates)

### PR DESCRIPTION
**Pull request by emmenko**
To fix some warnings and security updates:

```
warning license-checker-webpack-plugin > joi@14.3.1: This module has moved and is now available at @hapi/joi. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning license-checker-webpack-plugin > joi > topo@3.0.3: This module has moved and is now available at @hapi/topo. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning license-checker-webpack-plugin > joi > hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning license-checker-webpack-plugin > joi > topo > hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,30 +18,33 @@
   ],
   "main": "src/index.js",
   "scripts": {
+    "format": "npm run format:js && npm run format:md",
+    "format:js": "prettier --write '**/*.js'",
+    "format:md": "prettier --write --parser markdown '**/*.md'",
```
Comment (unindented) [inline package.json]: These lines seem unrelated to the goal of the pull request.

File: .prettierrc [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {
+  "parser": "babel",
```
Comment (unindented) [inline .prettierrc]: Why was this necessary?

Comment (emmenko) [reply .prettierrc]: As far as I remember it's a required option. I can double check though...

File: package.json (hunk 1 above)
Comment (emmenko) [reply package.json]: Yeah I can remove them if you want.

Comment (unindented) [reply package.json]: Yes please, they seem unrelated.

File: .prettierrc (hunk 2 above)
Comment (unindented) [reply .prettierrc]: I don't think it's required, cause it's working fine today.

**Comment by emmenko** [comment]
@unindented sorry for the delay. I removed the unnecessary changes, let me know if it's ok like this now. Thanks

**Comment by unindented** [comment]
Thank you!
